views/package.json

```diff
@@ -9,6 +9,6 @@
   },
   "dependencies": {
     "@oat-sa/tao-item-runner": "0.4.0",
-    "@oat-sa/tao-item-runner-qti": "github:oat-sa/tao-item-runner-qti-fe#fix/BOSA-36/text-entry-pattermask"
+    "@oat-sa/tao-item-runner-qti": "0.4.19"
```

Comment:
I don't get it.
The [`develop` branch](https://github.com/oat-sa/extension-tao-itemqti/blob/develop/views/package.json) seem to have it already, how come it shows that diff?

Reply:
Npm package was release to npm repository, so its better use this way. Apart of that, I deploy on playground and dont see the new code  (I dont know if its for this )

Reply:
I'll check the deployment for you, don't worry.